CMakeLists.txt

```diff
@@ -264,6 +253,28 @@ if (NOT EMSCRIPTEN)
     target_link_libraries(wasm-interp m)
   endif ()
 
+  # wastfmt
+  add_executable(wastfmt src/exe/wastfmt.c)
```

Comment:
Either of those would probably be OK with me. Maybe we should pick something that reflects the same language used where either the spec or design doc (when it exists) talks about sugar, or macros, or whatever. It looks like the spec just refers to "sugar" so maybe `wast-desugar` for now. When we introduce some text somewhere that describes the text format and its sugar, if we decide on a better term there, we can rename this to match.